Automattic/jetpack
# Publicize: refresh connections only when post is published

This PR refreshes the connections list once the post ~saves~ **publishes**. For this purpose, it adds a new `usePostJustPublished()` hook.

Fixes https://github.com/Automattic/jetpack/issues/21206

#### Changes proposed in this Pull Request:

Publicize: refresh connections only when post is published

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Create a new post.
* Open the Jetpack sidebar.
* Add social media connections.
* Enable some connections to share the post.
* Publish the post.


https://user-images.githubusercontent.com/77539/135130060-a343cc89-f92b-4245-9cd6-3dbf4856019b.mov


Confirm the connections list gets updated properly, only when the post is published. You can take a look at the network tab of the dev-tools of your dev client (Chrome, Firefox, Safari...)
